apps/explorer/package.json

```diff
@@ -17,9 +17,11 @@
         "preview": "vite preview",
         "storybook": "storybook dev",
         "build-storybook": "storybook build",
+        "pull-amplitude": "ampli pull web && node prependLicense.mjs && prettier -w ampli.json src/utils/ampli/index.ts",
```

Comment:
Plz don't roast me for this LOL (but more seriously, if you have better ideas here I'm open to them). Ampli disables eslint which means we have to manually prepend the Mysten headers on it (also, I think Prettier formatting the files doesn't hurt as opposed to just ignoring them)

@Jordan-Mysten